rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -3354,6 +3361,10 @@ public void setNodeTokens(String nodeUrl, List<String> tokens) throws RMExceptio
             registerAndEmitNodeEvent(rmNode.createNodeEvent(RMEventType.NODE_STATE_CHANGED,
                                                             rmNode.getState(),
                                                             rmNode.getProvider().getName()));
+
+            logger.info("The list of tokens of the node " + nodeUrl + "' are updated. The current list of tokens is: " +
```

Comment:
on the node is better?

Reply:
there is an extra single quote

Reply:
are updated => is updated

Reply:
Done except for 'on the node is better?'. the current sentence is simpler for me

Reply:
ok thx